docs/en/notes/contribution_guide.md

```diff
@@ -1 +1,136 @@
 # Contribution Guide
+
+OpenMMLab welcomes everyone who is interested in contributing to our projects and accepts contribution in the form of PR.
+
+## What is PR
+
+`PR` is the abbreviation of `Pull Request`. Here's the definition of `PR` in the [official document](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) of Github.
+
+```
+Pull requests let you tell others about changes you have pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.
+```
+
+## Basic Workflow
+
+1. Get the most recent codebase
+2. Checkout a new branch from `main` or `dev-1.x` branch, depending on the version of the codebase you want to contribute to (see [Maintenance Plan](../migration/overview.md) for more details)
+3. Commit your changes ([Don't forget to use pre-commit hooks!](#3-commit-your-changes))
+4. Push your changes and create a PR
+5. Discuss and review your code
+6. Merge your branch to `main` or `dev-1.x` branch
+
+## Procedures in detail
+
+### 1. Get the most recent codebase
+
+- When you work on your first PR
+
+  Fork the OpenMMLab repository: click the **fork** button at the top right corner of Github page
+  ![avatar](https://user-images.githubusercontent.com/22607038/195038780-06a46340-8376-4bde-a07f-2577f231a204.png)
+
+  Clone forked repository to local
+
+  ```bash
+  git clone git@github.com:XXX/mmocr.git
+  ```
+
+  Add source repository to upstream
+
+  ```bash
+  git remote add upstream git@github.com:open-mmlab/mmocr
+  ```
+
+- After your first PR
+
+  Checkout the latest branch of the local repository and pull the latest branch of the source repository. Here we assume that you are working on the `dev-1.x` branch.
+
+  ```bash
+  git checkout dev-1.x
+  git pull upstream dev-1.x
+  ```
+
+### 2. Checkout a new branch from the `main` branch or `dev-1.x` branch
+
+````bash
+
+```bash
+git checkout -b branchname
+````
+
+```{tip}
+To make commit history clear, we strongly recommend you checkout the `main`/`dev-1.x` branch before creating a new branch.
```

Comment:
```suggestion
To make commit history clear, we strongly recommend you checkout the `main` or `dev-1.x` branch before creating a new branch.
```

![image](https://user-images.githubusercontent.com/45810070/195239966-6cf56e15-3ec7-4a09-9fe6-f981a34516bb.png) not clear